### DISCUSSION TITLE
Where can I find more documentation on VSCode's LSP configurations and settings?

### DISCUSSION THREAD

**Discussion by kernelguardian** [microsoft/vscode-discussions#354] [Extension Development QnA] [answered]
I have been tinkering with the VSCode's Language Server Protocol for a while now. I really would like to learn more. However I couldn't find much resources. Right now I am trying to find the documentation for this part of code

`transport: TransportKind.ipc `

from [here](https://code.visualstudio.com/api/language-extensions/language-server-extension-guide#advanced-topics). What if I need to find other forms of transport communication like stdio or something else. What other communication options are there and possible examples. Where can I find these?

**Comment by burkeholland** [comment, marked as answer]
Per the doc you linked, this might be best raised as an issue on the LSP repo...

https://github.com/microsoft/language-server-protocol/issues

**Reply by kernelguardian** [reply]
Thank you @burkeholland
